.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,95 @@
-<!-- Please feel free to remove whatever sections/lines in this aren’t relevant.
+<!---
+Welcome! Thank you for contributing to the U.S. Web Design System. Your contributions are vital to our success and we are glad you're here.
 
-Use the title line as the title of your pull request, then delete these lines.
+Please keep in mind:
+- This pull request (PR) template exists to help speed up integration. The USWDS Core team reviews and approves every PR before merging it into the public code base, so the better we can understand the problem and solution, the sooner we can merge this change. The point here is: clear explanations matter!
+- You can erase any part of this template that doesn't apply to your pull request (including these instructions!).
 
-## Title line template: [Title]: Brief description
+If you get stuck, you can find more information about contributing in [contributing.md](https://github.com/uswds/uswds/blob/develop/CONTRIBUTING.md) or you can reach out to us directly at uswds@gsa.gov.
+ -->
 
-Website: For pull requests that impact designsystem.digital.gov’s look, feel, or functionality, please open a pull request on the uswds-site repo (https://github.com/uswds/uswds-site).
+<!---
+Step 1 - Title this PR with the following format:
+USWDS - [Package]: [Brief statement describing what this pull request solves]
+eg: "USWDS - Button: Increase font size"
+ -->
 
+## Summary
+_Provide a one or two sentence summary of the update that can be used in the changelog._
+<!--
+A successful summary is written in the past tense and includes:
+1. A benefit statement, and
+2. A description of the update
```

Comment:
The link to releases seems useful. A practical example might help or the points in the expected format. 

```suggestion
A successful summary is written in the past tense and includes:
**A benefit statement.** A description of the update.
```